lib/base/reset-normalize.less

```diff
@@ -69,7 +69,7 @@ main { /* 1 */
  */
 
 figure {
-  margin: 1em 40px;
+  margin: 1em calc(var(--su-static32) + var(--su-static8)); // 1em 40px
```

Comment:
Question for my understanding: why we don't store `1em` in a variable too? Are we just saying here use the same font size of your parent to the determine the vertical margin?

Side note for my benefits:
How do we decide when to use `px` over `em` over `rem`? Is there some historical document I can read about the decision?

Reply:
> why we don't store `1em` in a variable too?

I think we'd want an existing scale first. Prior to creating the `--su` variables, we had `@su` Less variables (see [PR](https://github.com/StackExchange/Stacks/pull/873#issue-1165843841)).

> Are we just saying here use the same font size of your parent to the determine the vertical margin?

Seems that way. This seems like a reasonable enough reset, but honestly it predates me at SO 🤷‍♂️

> How do we decide when to use `px` over `em` over `rem`?

`px` and static and `em`/`rem` are not 🤷‍♂️. Ignoring this one case, I think we've tended towards `px` values for greater control. For instance, if we used `em`/`rem` all over and we adjust the root font size, the sizing of many elements would be impacted. I think we tend towards `px` for the sake of control and I think that still persists even with these `--su` variables since we actively differentiate between ones that could actually be variable (`--su[x]`) and static ones (`--su-static[x]`).

With all that said, I think I know where this is headed 🙂. We _could_ use `rem` instead of `px` values then just change the base font size to adjust spacing (without the need for all of these spacing units). It's an interesting idea but probably one for us to shove back into the dark recesses of our mind until we have time to consider it completely.

> Is there some historical document I can read about the decision?

There's no historical Stacks document that I'm aware of that provides justification or context for usage of `em`/`rem`/`px`/etc 🤷‍♂️.

Reply:
Thanks for all the detailed replies. They helped. :)